src/os/layer/preset/presetmenubutton.js

```diff
@@ -35,12 +38,16 @@ const EventType = {
   APPLY_PRESET: 'apply-preset',
   REMOVE: 'remove',
   SAVE: 'save',
+  TOGGLE_PRESET: 'toggle-preset',
   TOGGLE_DEFAULT_TRUE: 'toggle-default-true',
   TOGGLE_DEFAULT_FALSE: 'toggle-default-false',
   TOGGLE_PUBLISHED_TRUE: 'toggle-published-true',
   TOGGLE_PUBLISHED_FALSE: 'toggle-published-false'
 };
 
+/**
+ * @const {GoogLog.Logger}
```

Comment:
I think this is mixing up `@type {...}` with `@const` which should be a standalone

Reply:
derp. ty ty